### PR TITLE
new resources/datasources: me_installation_template*

### PR DESCRIPTION
**Pull request by yanndegat**
- datasource/ovh_me_installation_templates
- datasource/ovh_me_installation_template
- resource/ovh_me_installation_template
- resource/ovh_me_installation_template_partition_scheme
- resource/ovh_me_installation_template_partition_scheme_hardware_raid
- resource/ovh_me_installation_template_partition_scheme_partition

### CODE REVIEW

**Comment by yanndegat** [comment]
/hold

missing:
- testaccs on updates
- importers + test
- docs

**Comment by yanndegat** [comment]
ping 
@gaotan
@pgaxatte

File: ovh/resource_ovh_me_installation_template_partition_scheme.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	}
+}
+
+func resourceMeInstallationTemplatePartitionSchemeCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	templateName := d.Get("template_name").(string)
+
+	opts := (&PartitionSchemeCreateOrUpdateOpts{}).FromResource(d)
+	endpoint := fmt.Sprintf("/me/installationTemplate/%s/partitionScheme", templateName)
```
Comment (pgaxatte) [inline ovh/resource_ovh_me_installation_template_partition_scheme.go]: Missing url.PathEscape

File: ovh/resource_ovh_me_installation_template.go [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+		},
+	}
+}
+
+func resourceMeInstallationTemplateCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	opts := (&InstallationTemplateCreateOpts{}).FromResource(d)
+
+	endpoint := fmt.Sprintf("/me/installationTemplate")
```
Comment (pgaxatte) [inline ovh/resource_ovh_me_installation_template.go]: fmt.Sprintf is not needed here

**Comment by pgaxatte** [comment]
Also you're missing the documentation ;)

**Pull request merged**